engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/node/ArrowCastFixedSizeNode.java

```diff
@@ -1,16 +0,0 @@
-package org.enso.interpreter.arrow.node;
-
-import com.oracle.truffle.api.nodes.Node;
-import org.enso.interpreter.arrow.LogicalLayout;
-import org.enso.interpreter.arrow.runtime.ArrowCastToFixedSizeArrayFactory;
-
-public class ArrowCastFixedSizeNode extends Node {
-
-  static ArrowCastFixedSizeNode create() {
-    return new ArrowCastFixedSizeNode();
-  }
-
-  public Object execute(LogicalLayout layoutType) {
-    return new ArrowCastToFixedSizeArrayFactory(layoutType);
```

Comment:
I am not sure what is a value of having a `Node` that invokes a constructor, @hubertp?

Reply:
The nodes are only useful for _self optimizations_ of the program. There is no _optimization_ in calling  `new ArrowCastToFixedSizeArrayFactory(layoutType)` - e.g. wrapping that into `Node` will not speed anything up. Only, if you'd like to _speculate on `layoutType` value_, but there is little sense in that when we are parsing at the same time - e.g. we cannot run at full speed anyway.